basictracer/scope.py

```diff
@@ -23,33 +23,28 @@
 from opentracing import Scope
 
 
-class BasicScope(Scope):
+class ThreadLocalScope(Scope):
     """BasicScope is the implementation of `opentracing.Scope`"""
 
-    def __init__(self, manager, span, finish_on_close=True):
+    def __init__(self, manager, span, finish_on_close):
         """Initialize a `Scope` for the given `Span` object
 
         :param span: the `Span` used for this `Scope`
         :param finish_on_close: whether span should automatically be
             finished when `Scope#close()` is called
         """
-        self._manager = manager
-        self._span = span
+        super(ThreadLocalScope, self).__init__(manager, span)
         self._finish_on_close = finish_on_close
-        self._to_restore = manager.active()
-
-    def span(self):
-        """Return the `Span` that's been scoped by this `Scope`."""
-        return self._span
+        self._to_restore = manager.active
 
     def close(self):
         """Finish the `Span` when the `Scope` context ends, unless
         `finish_on_close` has been set.
         """
-        if self._manager.active() is not self:
+        if self.manager.active is not self:
```

Comment:
At the time of writing, if we inherit from `Scope`, it should be `_manager` no? https://github.com/opentracing/opentracing-python/pull/64/files#diff-3e51a9eeb61cdae78a2c08ad32d1a78bR39